### PR TITLE
Make height and width optional and remove orientation

### PR DESCRIPTION
**Pull request by tkgnm**
## What does this change?

See [this ticket](https://theguardian.atlassian.net/browse/LIVE-5237)

I asked @guardian-ben if we need to provide width/height/orientation. 

”I don't think we do anymore. Alex Breuer changed to a fixed aspect ratio for all images that are hard-coded locally. However, I think we should keep them  but make them optional (except orientation, which can be deleted and calculated locally) in case we want a new card that shows the full image in the future” 

This PR ensures the correct change to the schema

### CODE REVIEW

File: proto/collection.proto [hunk 1]
```diff
@@ -39,13 +39,13 @@ enum MediaType {
 }
 
 message Image {
+    reserved 5
```
[the remark this answers is not in the document]
Comment (DavidLawes) [reply proto/collection.proto:42]: This is a great idea, but given we're not in production yet I think we can change/break the schema up until that point (and maybe it's advantageous to do so if it keeps it clean/simple)

[the remark this answers is not in the document]
Comment (tkgnm) [reply proto/collection.proto:42]: cool I'll remove that then :)

[the remark this answers is not in the document]
Comment (guardian-ben) [reply proto/collection.proto]: Yeah, we don't need to reserve this right now; as David said, we aren't in production, so it's not necessary yet. But this is exactly what to do when we are removing things from a production object.

**Review by DavidLawes — APPROVED** [review]
👍

**Pull request merged**